reactive/src/main/java/com/scmspain/howtospring/reactive/Endpoint.java

```diff
@@ -17,12 +22,17 @@ public Endpoint(MyService myService) {
     }
 
     @RequestMapping(path = "/users", method = RequestMethod.GET)
-    public Observable<User> listUsers() {
-        return myService.listUsers();
+    public Single<List<User>> listUsers() {
+        return myService.listUsers().toList().toSingle();
     }
 
     @RequestMapping(path = "/users/{userId}", method = RequestMethod.GET)
-    public Observable<User> showUser(@PathVariable String userId) {
-        return myService.showUser(userId);
+    public Single<User> showUser(@PathVariable String userId) {
+        return myService.showUser(userId).toSingle();
+    }
+
+    @RequestMapping(path = "/users.stream", method = RequestMethod.GET)
```

Comment:
I know it's not a correct endpoint, but I didn't wanted to mess up <code>listUsers</code> action